packages/aws_bedrock/data_stream/invocation/elasticsearch/ingest_pipeline/default.yml

```
@@ -284,6 +284,38 @@ processors:
       field: gen_ai.response.id
       copy_from: aws_bedrock.invocation.output.output_body_json.id
       ignore_failure: true
+  - script:
+      tag: append_input_message_content_kinds
+      if: ctx.aws_bedrock?.invocation?.input?.input_body_json?.messages != null && ctx.aws_bedrock.invocation.input.input_body_json.messages instanceof List
```

```suggestion
      if: ctx.aws_bedrock?.invocation?.input?.input_body_json?.messages instanceof List
```

It's okay to do `instanceof` on `null`.